jdaviz/core/template_mixin.py

```diff
@@ -4815,6 +4815,7 @@ class Table(PluginSubcomponent):
     # NOTE: These UI features are not covered in test coverage. Plugins making use of
     # this feature should ensure test coverage for their respective tables.
     show_rowselect = Bool(False).tag(sync=True)  # Flag to enable row selection boxes
+    single_select = Bool(False).tag(sync=True)  # Flag to limit selection to single entry
```

Comment:
would it be confusing to have this as `multiselect` (that defaults to True) instead to match what we use for dropdown selects?

Reply:
Good idea for consistency! Done in https://github.com/spacetelescope/jdaviz/pull/3499/commits/abf4a635782b2c47bb3c7831e57e0546576573a0